### ISSUE TITLE
provide properties that allow navigation button styles to be augmented, similar to how additionalClass works

### ISSUE THREAD

**Issue by dev6rodn** [xiaolin/react-image-gallery#455]
The additionalClass property only applies to the root element. It would be really helpful to be able to override the styles applied to the left nav, right nav and full screen buttons.

**Comment by xiaolin** [comment]
left nav, right nav and full screen buttons all have classNames you can override in your own css file

**Comment by dev6rodn** [comment]
Agreed. I was just wondering why this special property exists _only_ for the root node. I'll close this issue as it isn't a high priority.
